hasher-matcher-actioner/src/OpenMediaMatch/tests/test_api_banks.py

```diff
@@ -288,3 +288,28 @@ def test_banks_add_hash_index(app: Flask, client: FlaskClient):
     )
     assert post_response.status_code == 200
     assert post_response.json == {"matches": [2]}
+
+
+def test_bank_get_content_with_signals(client: FlaskClient):
+    bank_name = "TEST_BANK_GET_SIGS"
+    create_bank(client, bank_name)
+
+    image_url = "https://github.com/facebook/ThreatExchange/blob/main/pdq/data/bridge-mods/aaa-orig.jpg?raw=true"
```

Comment:
nit: This unittest will fail when run without internet access, but it looks like that is what all the other tests are doing, so /shrug

Reply:
Would it be worth us opening an issue to refactor tests to not use files from URLs but instead files from fixtures, and if we do need to test from URL we start up a local http file server (e.g., `python -m http.server -b 127.0.0.1 4000`) for that file URL?